futurenet/soroban-rpc/etc/soroban-rpc.env

```diff
@@ -2,6 +2,7 @@
 
 export ENDPOINT=localhost:8003
 export HORIZON_URL=http://localhost:8001
+export FRIENDBOT_URL=https://friendbot-futurenet.stellar.org/
```

Comment:
Is there a way for us to do same for futurenet horizon?

Reply:
So that anyone hitting the friendbot endpoint on the local horizon is sent to the futurenet one. Makes it so tests someone is running using an sdk can be agnostic to standalone vs futurenet vs testnet.

Reply:
Probably. I don't know enough about quickstart to say how off the top of my head. @sreuland ?

Reply:
it could be done, but maybe we address that as a new feature ticket - 'auto wire firendbot url for all network types' ? to let this go through for short term. it looks like we could add:

`export FRIENDBOT_URL="__FRIENDBOT_URL__"` in common/horizon/etc/horizon.env, and then add substitution of that token in `start` per network chosen, same as PASSPHRASE and ARCHIVE_URLS is done.

Reply:
+1 we don't need to do this now. Seeing this change made me think of it.